bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -167,15 +241,10 @@ def handle_bug(self, bug, data):
 
         # Sometimes a resolution comment is not provided so use a default
         bug_history["needinfo"] = False
-        bug_history["resolution_comment"] = "N/A"
-        for comment in bug_comments[::-1]:
-            if (
-                comment["creation_time"] == bug_history["status_time"]
-                and comment["author"] == bug_history["status_author"]
-            ):
-                bug_history["resolution_comment"] = comment["text"]
-                break
-        else:
+        bug_history["resolution_comment"] = self.get_resolution_comment(
+            bug_comments, bug_history
+        )
+        if bug_history["resolution_comment"] == "N/A":
```

Comment:
If it were to return `None` instead of `N/A`, we could have it as follows:
```suggestion
        resolution_comment = self.get_resolution_comment(bug_comments, bug_history)
        bug_history["resolution_comment"] = resolution_comment if resolution_comment else "N/A"

        if not resolution_comment:
```

Reply:
Instead of having a intermediate variable I left the resolution_comment setting the same, and changed the check to `... is None` and set the `N/A` comment there.